Vic2ToHoI4/Source/OutHoi4/OutLocalisation.cpp

```diff
@@ -0,0 +1,77 @@
+#include "OutLocalisation.h"
+#include "../Configuration.h"
+#include "Log.h"
+#include "OSCompatibilityLayer.h"
+#include <fstream>
+
+
+
+void outputLocalisations(const std::string& filenameStart, const languageToLocalisationsMap& localisations)
+{
+	for (const auto& languageToLocalisations: localisations)
+	{
+		if (languageToLocalisations.first == "")
+		{
+			continue;
+		}
+		std::ofstream localisationFile(filenameStart + languageToLocalisations.first + ".yml", std::ios_base::app);
+		if (!localisationFile.is_open())
+		{
+			throw std::runtime_error("Could not update localisation text file");
+		}
+		localisationFile << "\xEF\xBB\xBF"; // output a BOM to make HoI4 happy
```

Comment:
Does every utf8 file always need a bom in hoi4/imperator? Maybe we could have a common function to open it (with appropriate fs::utf8(path) name), paste BOM, and pass the handle back to main code?

Reply:
Opened issue in common items (which won't help me now, there's so much I need to do to be ready for the updates you've added), but can be added in the future.